### ISSUE TITLE
Middleman build error when posts files are not commited

### ISSUE THREAD

**Issue by it-praktyk** [AndrewKvalheim/middleman-vcs-time#3]
Hi
I use middleman-vcs-time with the latest Middleman.

When posts files are not added and commited to the repository 

```
[user@host]$ git status
On branch master
Your branch is ahead of 'origin/master' by 4 commits.
  (use "git push" to publish your local commits)
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

    modified:   source/posts/2016-05-29-Pester-related-links.html.md
    modified:   source/posts/2016-06-11-DSC-related-links.html.md

no changes added to commit (use "git add" and/or "git commit -a")
```

Under building site errors for creation of target html files are displayed - the partial output below

```
<CUT>
  identical  build/2016/05/09/how-dont-mock-Exchange-objects/index.html
  identical  build/2016/05/18/Using-OffCAT-from-a-command-line/index.html
  error  build/2016/06/11/DSC-related-links/index.html
   identical  build/posts/2016-05-09-Get-ExchangeServer-Members.txt
   identical  build/2016/05/17/Exchange-2013-Cumulative-Update-Installation-Fails/index.html
   identical  build/by-year/index.html
  error  build/index.html
   identical  build/javascripts/all-502ecede.js
  error  build/2016/05/29/Pester-related-links/index.html
  error  build/feed.xml
  identical  build/404/index.html
<CUT>
  remove  build/archives/index.html.gz
  gzip  Total gzip savings: 0 Bytes
  There were errors during this build, re-run with `middleman build --verbose` to see the full exception.
```

On the same tiime local middleman server display an error

```
NoMethodError at /
super: no superclass method `mtime' for #<Middleman::Sitemap::Resource:0x007ff97e662130>
Ruby    /home/wojtek/.gem/ruby/gems/middleman-vcs-time-0.0.4/lib/middleman-vcs-time/systems/git.rb: in mtime, line 10
Web     GET localhost/
```

**Comment by AndrewKvalheim** [comment]
Thanks for the report. I'd made a mistake that wasn't caught by the tests. It should be fixed in version 0.0.5.

**Comment by it-praktyk** [comment]
Thank you :)
